doozer

```diff
@@ -707,19 +763,23 @@ def images_build_image(runtime, odcs, repo_type, repo, push_to_defaults, push_to
         runtime.logger.info("No repos specified. --repo-type or --repo is required.")
         exit(1)
 
+    if not runtime.local:
+        threads = None
+
     results = runtime.parallel_exec(
         lambda (dgr, terminate_event): dgr.build_container(
             odcs, repo_type, repo, push_to_defaults, additional_registries=push_to,
-            terminate_event=terminate_event, scratch=scratch),
-        items)
+            terminate_event=terminate_event, scratch=scratch, realtime=(threads == 1)),
+        items, n_threads=threads)
```

Comment:
An idea for _the future_ would be to consider using the hacky progress bar I created for parallel exec stuff. Akin to this which still remains in elliott

https://github.com/openshift/elliott/blob/master/elliott#L478-L480

Reply:
OOoo nice. I was unaware of that. Definitely.